### PR TITLE
[Enhancement] added coordinate to tooltip export configuration

### LINKED ISSUE #846: [Bug] Interactions: Coordinate setting is not saved to the Map Config
**Describe the bug**
The state of the Coordinate setting under Interactions is not saved in the Map Config.  The Tooltip and Brush settings are saved, but the Coordinate is not.

**To Reproduce**
1. Enable Coordinate under the Interactions settings
2. Select Share and Export Map
3. Select the JSON format
4. Look at the Map Config
5. Under "interactionConfig" the only saved settings are for "tooltip" and "brush".  There is nothing related to Coordinate.

Tested using Kepler.gl version 1.1.11 on https://kepler.gl/demo/

**Expected behavior**
The Coordinate setting should be saved to the Map Config like the other Interaction Config settings and that Coordinate setting should be likewise be applied when loading a map from a JSON Map Config.

**Related Issue**
#504

### PR DESCRIPTION
**Pull request by macrigiuseppe**
Fixes #846 
Signed-off-by: Giuseppe Macri <gmacri@uber.com>

### CODE REVIEW

File: src/styles/base.js [hunk 1]
```diff
@@ -206,7 +206,7 @@ export const modalPadding = '10px 0';
 export const modalLateralPadding = '72px';
 export const modalPortableLateralPadding = '36px';
 
-export const modalOverLayZ = 1000;
+export const modalOverLayZ = 1001;
```
Comment (macrigiuseppe) [inline src/styles/base.js]: this makes sure the export modal dialog is fully visible when the tooltip is also active

File: src/reducers/vis-state-merger.js [hunk 2]
```diff
@@ -213,13 +213,15 @@ export function mergeInteractions(state, interactionToBeMerged) {
...
         ...state.interactionConfig[key],
         enabled,
-        config: pick(
-          {
-            ...state.interactionConfig[key].config,
-            ...configToMerge
-          },
-          Object.keys(state.interactionConfig[key].config)
-        )
+        ...(state.interactionConfig[key].config ? {
```
Comment (macrigiuseppe) [inline src/reducers/vis-state-merger.js]: coordinate entry doesn't have a config property

File: src/tasks/tasks.js [hunk 3]
```diff
@@ -24,7 +24,7 @@ import console from 'global/console';
 
 export const LOAD_FILE_TASK = taskCreator(
   ({fileBlob, info, handler, processor}, success, error) => {
-    console.time('loadFileTask')
+    console.time('loadFileTask');
```
Comment (heshan0131) [inline src/tasks/tasks.js]: delete

File: src/tasks/tasks.js [hunk 4]
```diff
@@ -36,7 +36,7 @@ export const LOAD_FILE_TASK = taskCreator(
           // result has both datasets and info
           // TODO: I think we should pass info to the handler and return
           // the same format back from the file handler
-          console.timeEnd('loadFileTask')
+          console.timeEnd('loadFileTask');
```
Comment (heshan0131) [inline src/tasks/tasks.js]: delete

File: test/node/schemas/vis-state-schema-test.js [hunk 5]
```diff
@@ -143,7 +143,8 @@ test('#visStateSchema -> v1 -> save load interaction', t => {
     brush: {
       enabled: false,
       size: 0.5
-    }
+    },
+    coordinate: { enabled: false }
```
Comment (heshan0131) [inline test/node/schemas/vis-state-schema-test.js]: remove space in brackats

File: src/reducers/vis-state-merger.js (hunk 2 above)
Comment (heshan0131) [reply src/reducers/vis-state-merger.js]: can you add tests in vis-schema to test for
1. coordinates are saved and correctly loaded
2. older config without coordinates will result in loading error

Comment (macrigiuseppe) [reply src/reducers/vis-state-merger.js]: @heshan0131 are you sure old configuration should fail? Would it be better if we handle them gracefully and default to __coordinate enabled: false__?

Comment (macrigiuseppe) [reply src/reducers/vis-state-merger.js]: let's make sure old configs are not triggering loading errors

**Pull request merged**